src/pyOpenMS/pyopenms/plotting.py

```diff
@@ -218,7 +212,7 @@ def plot_chromatogram(c: "MSChromatogram"):
 
 def _annotate_ion(mz: float, intensity: float, annotation: Optional[str],
                   color_ions: bool, annotate_ions: bool, matched: Optional[bool],
-                  annotation_kws: Dict[str, object], ax) -> Tuple[str, int]:
+                  annotation_kws: Dict[str, object], colormap: Dict[str, str], ax) -> Tuple[str, int]:
```

Comment:
Maybe a short description about the color map is needed in the following comment section.